serverlessworkflow/modules/ROOT/pages/cloud/index.adoc

```diff
@@ -0,0 +1,68 @@
+= {product_name} applications in Kubernetes
+:compat-mode!:
+// Metadata:
+:description: Workflow applications in Kubernetes
+:keywords: cloud, kubernetes, docker, image, podman, openshift, pipelines
+// other
+
+{product_name} is evolving as platform to build and deploy workflow applications in Kubernetes clusters.
+
+You have two options to deploy your workflow applications. Either as a full Quarkus project, or using the {operator_name}.
+
+The cards below list every feature included in the platform to deploy workflow applications either with Quarkus or the Operator.
+
+[NOTE]
+====
+Eventually these two options will converge, the {operator_name} will also be able to handle full Quarkus projects. So if you opt in to use Quarkus now and manually deploy your workflows, bear in mind that it's on the project's roadmap to integrate the Quarkus experience with the Operator.
+====
+
+[.card-section]
+== Kubernetes with Quarkus
+
+For Java developers, you can use Quarkus and a few add-ons to help you build and deploy the application in a Kubernetes cluster. {product_name} also generates basic Kubernetes objects YAML files to help you getting started. The application should be managed by a Kubernetes administrator.
+
+[.card]
+--
+[.card-title]
+xref:cloud/quarkus/build-workflow-image-with-quarkus-cli.adoc[Building workflow images using Quarkus CLI]
+[.card-description]
+Learn how to build images for your workflow applications using Quarkus CLI
+--
+
+[.card]
+--
+[.card-title]
+xref:cloud/quarkus/kubernetes-service-discovery.adoc[Kubernetes service discovery in {product_name}]
+[.card-description]
+Learn what is and how the Kubernetes service discovery for workflow application configuration works
+--
+
+[.card]
+--
+[.card-title]
+xref:cloud/quarkus/deploying-on-minikube.adoc[Deploying your {product_name} application on Minikube]
+[.card-description]
+Learn how to deploy your workflow application on Minikube for local tests and development
+--
+
+[.card]
+--
+[.card-title]
+xref:cloud/quarkus/deploying-on-kubernetes.adoc[Deploying your {product_name} application on Kubernetes]
+[.card-description]
+Learn how to deploy your workflow application on Kubernetes
+--
+
+[.card-section]
+== Kubernetes with the Operator
+
+For developers that are looking for a native Kubernetes approach where you can model workflows using YAML definitions and directly deploy them, you can use the {operator_name}. The operator registers a new Kubernetes resource in the cluster to manage your workflow development iteration cycle and composition of services and events. The application is managed by the operator.
```

Comment:
is this implemented already?
Or are you referring to the online editor that connects to the k8s instance?

Reply:
I don't follow. It's implemented with the current snapshot. One can create a workflow YAML and deploy it on k8s.

Reply:
I meant, because you added you mentioned "you cam model". isn't it part of the online editor?

Reply:
No, you can model workflows using YAML :P